server/src/database/typeorm-seeds/metadata/field-metadata/favorite.ts

```diff
@@ -135,7 +135,7 @@ export const seedFavoriteFieldMetadata = async (
         targetColumnMap: {},
         description: 'Favorite workspace member',
         icon: 'IconCircleUser',
-        isNullable: false,
+        isNullable: true,
```

Comment:
I think this one should stay not Nullable

Reply:
It throws an error if not nullable